.travis.yml

```diff
@@ -54,6 +44,16 @@ matrix:
             - pandoc
             - graphviz
 
+#    - os: linux
+#      dist: bionic
+#      language: python
+#      python: 3.9
+#      addons:
+#        apt:
+#          packages:
+#            - pandoc
+#            - graphviz
+
```

Comment:
why leaving this commented?

Reply:
I'll remove it. python 3.9 does not work yet